CSE437s/FL24-project-group-9
# Frontend - Integrate API

# Description

- Integrating frontend with backend localhost API endpoints
- Add PlannerPageV2 to test out new design

## Related PRs

N/A

## Type of change

### Please delete options that are not relevant


## Screenshots/Recordings


## Testing instructions

Instructions on how to test the changes made in the PR, helping reviewers validate the code.

1. Run `git checkout ngoctramvu/integrate-api`.
2. Run `npm install` and `npm run dev`.
3. Navigate through user flow
4. Verify that functionality planner works as expected.

## Notes
